gemma-core/src/main/resources/ubic/gemma/model/common/description/Characteristic.hbm.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
-        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+    "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 
 <hibernate-mapping>
-    <class name="ubic.gemma.model.common.description.Characteristic" table="CHARACTERISTIC"
-           discriminator-value="Characteristic">
```

Comment:
I really don't know why this would have a discriminator. By removing this, it will use `NULL` which is a sensible default for an unspecialized characteristic.